sdk/src/main/java/com/mapbox/maps/renderer/SurfaceUtils.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.maps.renderer
+
+import android.graphics.Bitmap
+import android.graphics.Rect
+import android.util.Log
+import android.view.Surface
+import java.lang.Exception
+
+fun Surface.drawWidget(bitmap: Bitmap) {
```

Comment:
is it a leftover? do not see it being called anywhere

Reply:
Removed